include/bmp.h

```diff
@@ -0,0 +1 @@
+void hello(void);
```

Comment:
Header files require include guards (https://en.wikipedia.org/wiki/Include_guard#Example). Read that to understand why.

Basically you just need to start ever .h file with:
```
#ifndef YOUR_FILE_NAME_H
#define YOUR_FILE_NAME_H
```

And finish it with
`#endif // YOUR_FILE_NAME_H`